### PR TITLE
Enable the Utility Service in BLE Pairing Mode.

### PR DESCRIPTION
**Pull request by microbit-carlos**
This service enables datalog access to the apps, to be able to view the logged data without a computer.

This flag enables the service in pairing mode only and should have no effect when running MicroPython.

More info in:
- https://github.com/lancaster-university/codal-microbit-v2/issues/354

### CODE REVIEW

**Comment by microbit-carlos** [comment]
Okay, I've tested the CI builds and can access the log from the iOS app 🎉

**Comment by dpgeorge** [comment]
This looks fine to me, I can't think how it would interfere with MicroPython, which only accesses the log through CODAL.

Note: I have not tested this PR.

File: src/codal_app/codal.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
         "DEVICE_STACK_SIZE": 8192,
         "MICROBIT_BLE_ENABLED" : 0,
         "MICROBIT_BLE_PAIRING_MODE": 1,
+        "MICROBIT_BLE_UTILITY_SERVICE_PAIRING": 1,
```
Comment (dpgeorge) [inline src/codal_app/codal.json:13]: Should this go in alphabetical order, ie after `MICROBIT_BLE_SECURITY_MODE`?

Comment (microbit-carlos) [reply src/codal_app/codal.json]: Yeah, I placed it there to be next to the other pairing mode config option, but make sense to move it alphabetically 👍

**Pull request merged**